src/style.css

```diff
@@ -1,3 +1,86 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
   margin: 0;
+  padding: 0;
+  border: 0;
+  display: block;
+  }
+
+li {
+  display: inline;
+  height: 60px;
+  margin: 0 20px;
+  color: #000;
+  font-family: sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+li:hover {
+  border-bottom: 2px solid #00ACDC;
+  color: inherit;
+}
+
+li.is-active::after {
+  display: block;
+  width: 100%;
+  height: 2px;
+  background-color: #00ACDC;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.blue1 {
+  color: #00ACDC;
+}
+
+li.is-active {
```

Comment:
The is-active class for li elements has pointer-events set to none, which means the li element will not react on click events. This might not be the desired behavior for a navigation element.